libraries/ms-helper/test/test_thermistor.c

```diff
@@ -1,4 +1,7 @@
+#include "gpio.h"
+#include "interrupt.h"
 #include "log.h"
+#include "soft_timer.h"
```

Comment:
Why is soft timer included here?

Reply:
If it's unnecessary I can remove it. I don't actually know much about soft timers, do we have a document on it?

Reply:
You shouldn't include things you aren't using. The code is the best documentation for soft timers.